examples/aws/verify_egressv1.go

```diff
@@ -0,0 +1,50 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	ocmlog "github.com/openshift-online/ocm-sdk-go/logging"
+	"github.com/openshift/osd-network-verifier/pkg/cloudclient"
+)
+
+// Example config struct
+type egressConfigV1 struct {
+	debug bool
+}
+
+func extendValidateEgressV1(ctx context.Context) error {
+	//---------Initialize required args---------
+	// Read AWS creds from environment
+	key, _ := os.LookupEnv("AWS_ACCESS_KEY_ID")
+	secret, _ := os.LookupEnv("AWS_SECRET_ACCESS_KEY")
+	session, _ := os.LookupEnv("AWS_SESSION_TOKEN")
+	// Build the v1 credentials
+	creds := credentials.NewStaticCredentials(key, secret, session)
+	builder := ocmlog.NewStdLoggerBuilder()
```

Comment:
consider using `ocmlog.NewStdLoggerBuilder().Debug(true).Build()` to simplify

Reply:
I'd say the current form is looking a bit complex, and it felt like we're implifying the caller that it should be in this form

Reply:
> consider using `ocmlog.NewStdLoggerBuilder().Debug(true).Build()` to simplify

Thanks, I was looking for a simpler way to create logger! Updated to use `ocmlog.NewStdLoggerBuilder().Debug(true).Build()`

Reply:
> I'd say the current form is looking a bit complex, and it felt like we're implying the caller that it should be in this form

Are you referring to the logger?